packages/arquero-react/src/ArqueroDetailsList/__tests__/RowGroupingTestStory/RowGroupingTestStory.styles.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+import styled from 'styled-components'
+
+export const Table = styled.div`
+	margin-top: 12px;
+	height: calc(100vh - 220px);
```

Comment:
You may want to consider starting a shared styles file so you don't need to repeat these.